README.md

```diff
@@ -399,7 +422,7 @@ Of all the entity types, two of them contain sensitive information which is neve
 - Keys (Keystore entries used for example by listeners)
 - Secrets (Secure Passwords and SSH keys)
 
-A encryption passphrase provided by the graphman requester is used to encrypt and decrype these secrets. This
+A encryption passphrase provided by the graphman requester is used to encrypt and decrypt these secrets. This
 encryption passphrase is set with HTTP header name `l7-passphrase`. If missing, graphman will use its local master
```

Comment:
should we change this to x-l7-passphrase ?

Reply:
As it is from the postman collection, I just ignored for now.